packages/loader/container-loader/src/loader.ts

```diff
@@ -51,53 +51,30 @@ function canUseCache(request: IRequest): boolean {
 }
 
 export class RelativeLoader extends EventEmitter implements ILoader {
-    // Because the loader is passed to the container during construction we need to resolve the target container
-    // after construction.
-    private readonly containerDeferred = new Deferred<Container>();
-
     /**
      * BaseRequest is the original request that triggered the load. This URL is used in case credentials need
```

Comment:
It says the original request is used in case credentials needs to be fetched again? I don't understand what this means here? The tokens are provided by hosts and refreshed in case they are expired.

Reply:
i don't think the comment is relevant anymore

Reply:
Yes, i think so too.